src/V1/APIs/Config/migrations/20221028082513_users.ts

```diff
@@ -0,0 +1,20 @@
+import { Knex } from 'knex';
+import { UserType } from '../../Models/User';
+
+export async function up(knex: Knex): Promise<void> {
+    return knex.schema.createTable('users', (table) => {
+        table.increments('id').primary();
+        table.string('first_name', 50).notNullable();
+        table.string('last_name', 50).notNullable();
+        table.string('password_digest', 100).notNullable();
+        table.bigInteger('phone_number').notNullable();
+        table.string('email', 100).unique().notNullable();
+        table.integer('verification_code', 10).notNullable();
```

Comment:
This might create a problem...It can't always have value you know?

Reply:
Do you mean I should remove the notNullable contrainst? @forestbaba